### ISSUE TITLE
Confusion over UserAgentOverride locale and platform

### ISSUE THREAD

**Issue by kekearif** [berstend/puppeteer-extra#345]
For the `UserAgentOverride` what exactly is `locale`? Do these languages have to match the browser languages set by puppeteer? For example if I go to `https://bot.sannysoft.com` it will say my languages are `en-US,en`. Are these the values I should use? What is `q=0.9`?

This is a user agent I found `Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.36`, how would I construct the object for this?

my guess is something like this:

```
const ua = UserAgentOverride({
  userAgent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/70.0.3538.102 Safari/537.3',
  locale: 'en-US,en;q=0.9',
  // Is this right for the platform?
  platform: 'macOS'
})
```

but I'm not sure if that's correct or not.

**Comment by berstend** [comment]
Uhm, this was a documentation bug once (#132) that was fixed: https://github.com/berstend/puppeteer-extra/pull/276/files#diff-2986a3762c2789dc3a2423dfc2a25fbe

Seems like this reverted for some reason, will fix this in the next days.

Until then, this is the correct way to set the locale:
```js
const puppeteer = require('puppeteer-extra')
const StealthPlugin = require('puppeteer-extra-plugin-stealth')
const stealth = StealthPlugin()
// Remove this specific stealth plugin from the default set
stealth.enabledEvasions.delete('user-agent-override')
puppeteer.use(stealth)
// Stealth plugins are just regular `puppeteer-extra` plugins and can be added as such
const UserAgentOverride = require('puppeteer-extra-plugin-stealth/evasions/user-agent-override')
// Define custom UA, locale and platform
const ua = UserAgentOverride({
  userAgent: 'Mozilla/4.0 (compatible; MSIE 6.0; Windows NT 5.1; SV1)',
  locale: 'de-DE,de',
  platform: 'Win32'
})
puppeteer.use(ua)
```

The q stuff is called qvalues and defines a weight, which is not supported to be used through this API.

The underlying API being used is `Network.setUserAgentOverride`:
![image](https://user-images.githubusercontent.com/1368633/95016769-5b60f600-0655-11eb-8684-40d112465ee0.png)

Regarding the platform:
![image](https://user-images.githubusercontent.com/1368633/95016787-7cc1e200-0655-11eb-85d7-0663b068334c.png)

**Comment by kekearif** [comment]
Thanks for this! So in my example above I would have:

useragent:` Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_1) AppleWebKit/537.36 (KHTML, like Gecko)`
locale: `en-US,en`
platform: `MacIntel`

does this sound right?

**Comment by berstend** [comment]
Fixed in #303
